### ISSUE TITLE
Upgrade Blueprint to v0.9.1

### ISSUE THREAD

**Issue by chriseppstein** [Compass/compass#45]
Blueprint version 0.9.1 was recently released with the following changes:

```
9be68576 Fixing bugs, see Lighthouse for changes
414d9242 Add visibility:hidden to hr.space so it's useful outside of white backgrounds.
7cdabb82 Added PSD template to help with mockups
c2493689 css_parser now uses Array#join instead of Array#to_s for ruby1.9 compat
74d7ecf9 Change CSS append order to put custom CSS after plugin CSS--allowing custom CSS to over-ride plug-in properties.
4d61d5db Changed custom CSS comment to file name instead of file path--removing the need to sanatize the CSS comment for security/privacy reasons.
b881bf59 Fixed some typos in TUTORIAL.textile.
e9e91bb9 When splitting on a colon for key/value CSS pairs, use first colon as the only split Signed-off-by: Joshua Clayton <jclayton@fusionary.com>
eb58cdef Fix grid builder to properly generate png
0811b3d6 Allow for the specification of a non-default settings file via the command line in compress.rb
d77e914e Password inputs got the same style as regular text inputs.
0f7588a5 Cleaned up whitespace in changelog; updated version numbers within stylesheets
05312a80 Updating authors one last time
2dd0d5ed Updating contributors, fixing grid.css
7e63ed3e Cleaning forms.html
3bf83f71 Test form whitespace cleanup
bdd4568f Remove p tag
```

Compass needs to merge these changesets into it's port.

**Comment by chriseppstein** [comment]
This change should be part of the v0.10 release.

**Comment by chriseppstein** [comment]
016c9114f1aa91f84c3d68f3be9a97a114334d84
63795c8bfe31cbbfad726f714bf8d22770adc7ad

too

**Comment by chriseppstein** [comment]
fixed in v0.10
